src/index.ts

```diff
@@ -54,8 +54,7 @@ function localTimestamp() {
 }
 
 const localTimestampFormat = winston.format((info) => {
-    info.timestamp = localTimestamp();
-    return info;
+    return { ...info, timestamp: localTimestamp() };
```

Comment:
Thinking through this in my head: I think this function should be returning a winston format object and not a JSON object. The returned value gets combined with other winston formats to create an aggregate of all allowed formats. The JSON object won't be recognized as valid at that stage.